integration-tests/hive-hcatalog/test_hive.py

```diff
@@ -0,0 +1,86 @@
+import json
```

Comment:
Can we put the test files in the same directory as the init actions? E.g. `hive-hcatalog/test_hive.py` rather than in the integration-tests directory

Reply:
@karth295 Technically it can be moved, but not without implications. The way it's structured right now allows to:
- separate shell scripts of init actions and tests
- allow to run all tests at once or individual tests from the `integration-tests` directory directly

If I move `test_hive.py` to `hive-hcatalog` test code will be mixed with init-action shell script which may be confusing. Also, in order to be able to run the tests, I'd have to:
- add `__init__.py` file in `hive-hcatalog` (so add another python file, not directly related to init-action shell script)
- rename `integration-tests` into `integration_tests` to be able to import `DataprocTestCase` in `test_hive.py`

In my opinion, it's not worth to move. WDYT?

Reply:
Thanks for the detailed analysis!

Just to clarify: can you run all the tests together if you split them out by directory?

If you can, then I still think it's worth it because I'd like to make it clear to users and maintainers that you should modify the tests before opening a PR. I think it's okay that we're adding an empty python file in every directory.

If not (or if it isn't straightforward), then let's just leave it as is.